README.md

```diff
@@ -76,9 +110,14 @@ lazy val core = (project in file("core"))
   .settings(libraryDependencies += AkkaSerializationHelperPlugin.annotation)
 ```
 
-### 2. Dump Persistence Schema
+## Incompatibility of persistent data
+
+![Typical tragic story](docs/typical-tragic-story.png "Typical tragic story")
```

Comment:
Nitpick: doesn't the text in `[...]` already serve as alt-text? is the text in `"..."` also needed?